mindsdb/interfaces/storage/model_fs.py

```diff
@@ -207,3 +209,37 @@ def json_list(self):
 
     def json_del(self, name):
         ...
+
+    def export_files(self) -> bytes:
+        folder_path = self.folder_get('', not_empty=True)
+        if folder_path is None:
+            return None
+
+        # parent_folder = os.path.dirname(folder_path)
+
+        zip_fd = io.BytesIO()
+
+        with zipfile.ZipFile(zip_fd, 'w', zipfile.ZIP_DEFLATED) as zipf:
```

Comment:
will be good to lock dir, because files may be deleted while archiving. But let ignore it for now

Reply:
in our case it is temporal integration to export from and a new one to export to. so all of them shouldn't be used concurrently